server/controller/websocket.py

```diff
@@ -129,17 +123,15 @@ def log_dump(message, websocket, metadata):
         'logEntries': html_logs,
     }
 
-    for connection in associated_websockets:
+    for connection in _get_associated_websockets(metadata['apiKey']):
```

Comment:
Should use the get('apiKey', '') in case 'apiKey' key is not in the metadata dictionary.